src/firmware_new/boards/frankie_v1/BUILD

```diff
@@ -0,0 +1,139 @@
+load("//firmware_new:rules.bzl", "stm32h7_hal_library_files_genrule")
+
+cc_binary(
+    name = "frankie_v1_main.elf",
```

Comment:
Just to be clear, does this name _have_ to end in `.elf`? An `.elf` file will be generated regardless of the name right because of the toolchain?

Reply:
Nope, it will output an `.elf` file, but the full name will exactly match this name (so if `name = "frankie_v1_main"` then it would output a file called `frankie_v1_main`).

Reply:
A comment at least indicating this rule outputs an `.elf` file regardless of the name would be great.

Reply:
No just to be clear this has one output, and that output has *exactly* this name.

Reply:
Yes, but regardless of the name it's an `.elf` file right?

Reply:
Nope, it will _literally_ be this name, if the name is `a.elf` it will be `a.elf`. If the name is `a` it will be `a`.

Reply:
After offline discussion, I get what you mean here now.

Can you think of a not-confusing way to phrase this? Everything I can think of just seems more confusing.

Reply:
Maybe "This rule will generate an `.elf` file regardless of the name of the target. ie. even if this target was named `frankie_v1_main`, an `.elf` file would still be created.

Reply:
After offline discussion, just removed the `.elf` file extension from the name.